c_common/models/extra_monitor_support/extra_monitor_support.c

```diff
@@ -801,16 +807,23 @@ void write_missing_sdp_seq_nums_into_sdram(
         missing_sdp_seq_num_sdram_address[
             number_of_missing_seq_nums_in_sdram +
             (offset - start_offset)] = data[offset];
+        if (data[offset] > max_seq_num){
+            io_printf(IO_BUF, "storing some shitty seq num. WTF %d %d\n",
```

Comment:
Unprofessional language in a message that might (but hopefully won't! 😉) get seen by users